openml/tasks/task.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, task_id, task_type_id, task_type, data_set_id,
         self.target_name = target_name
         self.split = None
 
-    def get_X_and_y(self):
+    def get_X_and_y(self, dataset_format='array'):
         """Get data associated with the current task.
```

Comment:
type hint, doc string with legal values for `dataset_format` parameter.

Reply:
Done.